src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolSscsJudicialOfficeMappingTest.java

```diff
@@ -112,60 +63,8 @@ void shouldReturnTribunalMemberMedicalRolesNoAccessRequests() {
             }
         });
 
-        assertEquals("hearing_work,decision_making_work,routine_work,priority",
-                roleAssignments.get(3).getAttributes().get("workTypes").asText());
-    }
-
-    //=================================FEE-PAID ROLES==================================
```

Comment:
Rather than retired can this be either split or adjusted so JOH tests remain for :

- Tribunal Member Medical,Fee Paid
- Tribunal Member Disability,Fee Paid
- Tribunal Member Financially Qualified,Fee Paid
- Tribunal Member Lay,Fee Paid
- Tribunal Member Optometrist,Fee Paid
- Tribunal Member Service,Fee Paid
- Tribunal Member,Fee Paid
- Tribunal Judge,Fee Paid

Reply:
> added back in 74c13e5a5bb444258a1b9e740e0776ba34eec759. only removed work types check, as there is already a thorough work type check in `DroolSscsJudicialRoleMappingTest`